spec/defines/keepalived_vrrp_track_file_spec.rb

```diff
@@ -11,7 +11,7 @@
   on_supported_os.each do |os, facts|
     context "on #{os}" do
       let(:facts) do
-        facts
+        facts.merge({ 'keepalived_version' => '2.2.0' })
```

Comment:
can you extend the tests so they run with keepalived_version 2.2.0 and maybe keepalived_version 2.0.0?